scanpy/readwrite.py

```diff
@@ -332,6 +335,9 @@ def read_visium(
     path = Path(path)
     adata = read_10x_h5(path / count_file, genome=genome)
 
+    adata.uns["spatial"] = dict()
+    adata.uns["spatial"][library_id] = dict()
```

Comment:
To get library id (if it holds that it's always present)

```suggestion
    with h5py.File(path / count_file) as f:
        attrs = dict(f.attrs)
    if library_id is None:  # if None was the default value
        library_id = str(attrs.pop("library_id")[0])

    adata.uns["spatial"][library_id] = dict()
```